cord/user_client.py

```diff
@@ -27,3 +32,20 @@ def create_with_ssh_private_key(ssh_private_key: str, password: str = None, **kw
         querier = Querier(user_config)
 
         return CordUserClient(user_config, querier)
+
+    def create_project(self, project_title: str, dataset_hashes: List[str], project_description: str = "") -> str:
+        project = {
+            'title': project_title,
+            'description': project_description,
+            'dataset_hashes': dataset_hashes
+        }
+
+        return self.querier.basic_setter(Project, uid=None, payload=project)
+
+    def create_project_api_key(self, resource_id: str, scopes: List[APIKeyScopes]):
+        payload = {
+            'scopes': list(map(lambda scope: scope.value, scopes))
+        }
+
+        return self.querier.basic_setter(ProjectAPIKey, uid=resource_id, payload=payload)
+
```

Comment:
Add options to retrieve the ProjectAPI Keys

Reply:
done